.github/ISSUE_TEMPLATE/oncall-rotation.md

```diff
@@ -21,6 +21,15 @@ Resources:
 
 ## Release
 
+- [ ] Choose a new version number in the format `vX.Y.Z`. If any of the following apply to the release, bump the minor version (Y). Otherwise, bump the patch version (Z). Do not bump the major version (X) without discussing with #engineering, as this will need more extensive communication and planning.
+
+  - A brand new feature (feature flagged or not)
```

Comment:
**nitpick:** `(feature flagged or not)` maybe clarify that the minor version should be bumped when the feature is generally available as opposed to released behind a flag?

Reply:
> I suppose it is (in theory anyway) completely isolated and not user-visible when disabled when it's behind the feature flag

This is just as often not the reality though 😆. Maybe we should bump it both when the feature is introduced behind the flag and when the flag is removed? Minor versions are cheap.

Whatever we do just want it to be explicit here.